x-pack/plugins/security/server/authorization/actions/alerts.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isString } from 'lodash';
+
+export class AlertsActions {
+  private readonly prefix: string;
+
+  constructor(versionNumber: string, spaceId: string) {
+    this.prefix = `alerts:${spaceId}:${versionNumber}`;
+  }
+
+  public get(owner: string, operation: string): string {
+    if (!operation || !isString(operation)) {
```

Comment:
Are these checks necessary given the type signature of the function? The only case in which this would be true (and pass type check) is if you passed in an empty string.